README.md

```diff
@@ -5,4 +5,6 @@
 
 This is the repository for the Data Book for SWDB
 
+For instructions on building the databook, see [here](https://allenswdb.github.io/build-instructions.html).
```

Comment:
Super minor, but if for some reason someone needs to coldstart the databook, this page won't exist. Is this a scenario we want to accommodate?

Reply:
I think this is more the root of my comment above: I think all the info in the instructions belongs in the readme. If we think about what a readme does, it: introduces the project, how to install, how to build, how to develop, how to configure, and it lists other links and resources associated with the project. It also has the benefit of being semi-standardized and easy to find. I guess I don't see the benefit of putting this information in a hidden page in the databook. It is hard to find by design. It's obscured from general view by not being in the ToC, and basically the only way to find it is by following the link in the readme file. Why don't we just eliminate that step and put the info in the readme?